cmd/committee-cli/main.go

```
@@ -0,0 +1,128 @@
+	natsURL := env.Get("NATS_URL", "nats://localhost:4222")
+	client, err := nats.NewClient(ctx, nats.Config{
+		URL:           natsURL,
+		Timeout:       10 * time.Second,
+		MaxReconnect:  3,
+		ReconnectWait: 2 * time.Second,
+	})
+	if err != nil {
+		slog.ErrorContext(ctx, "failed to connect to NATS", "error", err, "url", natsURL)
+		os.Exit(1)
+	}
+	defer client.Close()
```

golangci-lint (2.12.1)

[error] Error return value of `client.Close` is not checked

(errcheck)